bam_plcmd.c

```diff
@@ -1338,7 +1054,7 @@ int bam_mpileup(int argc, char *argv[])
         {NULL, 0, NULL, 0}
     };
 
-    while ((c = getopt_long(argc, argv, "Agf:r:l:q:Q:uRC:BDSd:L:b:P:po:e:h:Im:F:EG:6OsVvxXt:aM",lopts,NULL)) >= 0) {
+    while ((c = getopt_long(argc, argv, "Agf:r:l:q:Q:uRC:Bd:b:o:EG:6OsVvxXaM",lopts,NULL)) >= 0) {
```

Comment:
Did you mean to leave `g…u…v` in here? They're gone from the switch cases, so they just fall through to “Unexpected global option. … Usage: …”.

I was guessing there might be going to be new `case` labels for these three, just printing out “These options have been removed. Blah blah…”.